### DISCUSSION TITLE
Connection between z2m and HA : KO on restart

### DISCUSSION THREAD

**Discussion by jerome6994** [Koenkk/zigbee2mqtt#20735] [General]
Good morning
I come to you to ask for your help.
For quite a few months now I've had a problem that I haven't been able to resolve.
I will describe my installation to you first:
HAOS (in its latest version) installed in VM under promox (also in its latest version)
I have a VM under proxmox with Docker, Portainer installed on it. And I have functional mosquitto and z2m containers with conbee 2
![image](https://github.com/Koenkk/zigbee2mqtt/assets/66921143/3d8e237b-4cbf-4aac-bd51-1ea8f85cc0a1)

My issue is the following :
When I restart HA to validate a configuration or even just like that everything goes well except on the HA <-> Mosquitto <-> z2m connection

To have my functional entities in HA I have to restart z2m each time.

We are looking at the subject with several friends. There are at least 2 of us in my group who have the same problem and 2 others who don't have the problem. We compare the configurations but we don't see anything special!

We are a little desperate on this subject.

HA replied to me that he had not touched the MQTT integration so that it did not come from them but from you!
I don't know how to say it, I don't have the skills for this, I'm just asking you for a little help.

Thank you in advance

Jerome
[log.txt](https://github.com/Koenkk/zigbee2mqtt/files/13903036/log.txt)

**Comment by djsyl** [comment]
hello, confirmation !
I have this problem.
